problem_builder/answer.py

```diff
@@ -53,6 +52,11 @@ class AnswerMixin(XBlockWithPreviewMixin, XBlockWithTranslationServiceMixin, Stu
     """
     Mixin to give an XBlock the ability to read/write data to the Answers DB table.
     """
+    USER_STATE_FIELDS = ['edxnotes_visibility', ]
```

Comment:
Please put a comment above: `# Define fields returned in student_view_user_state`